packages/theme/css/foundations/size.css

```diff
@@ -43,12 +40,12 @@
   --salt-size-indicator: 4px;
   --salt-size-selectable: 18px;
   --salt-size-bar-strong: 16px;
-  /* --salt-size-bar-small: 4px; */
```

Comment:
this token was added/commented out in the `theme-updates` branch, removing it in this PR as no longer applicable